vnet/spec/vnet/endpoints/1.0/route_links_spec.rb

```diff
@@ -32,7 +32,8 @@ def app
   end
 
   describe "PUT /:uuid" do
-    accepted_params = { :mac_address => "fe:17:9b:9f:e8:33" }
+    accepted_params = {
+    }
```

Comment:
And here too. There is no change to `endpoints/route_links` to match this. `:mac_address` is still an accepted parameter.